UI/window-basic-main.cpp

```diff
@@ -3205,6 +3206,37 @@ void OBSBasic::TimedCheckForUpdates()
 #endif
 }
 
+void OBSBasic::ShowReleaseNotes()
```

Comment:
As this doesn't actually show the release notes, a more appropriate name might be CheckReleaseNotes or FetchReleaseNotes.